opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -541,8 +573,10 @@ def train_model(
         )
 
         print("Preparing model to save...\n")
+        # openfile = open(output_model_path,'w')
```

Comment:
remove debug code?

Reply:
sure, looked through all # and removed the unnecessary lines